src/modules/commander/commander_params.c

```diff
@@ -352,6 +352,22 @@ PARAM_DEFINE_INT32(COM_ARM_SWISBTN, 0);
  */
 PARAM_DEFINE_INT32(COM_LOW_BAT_ACT, 0);
 
+/**
+ * Delay between battery state change and failsafe reaction
+ *
+ * Battery state requires action -> wait COM_BAT_ACT_T seconds for the user to realize and take custom action
```

Comment:
@MaEtUgR Do you want to add a comment that it will switch to hold mode (or loiter)?

Reply:
Done in 28ab825